pages.pt-BR/common/ab.md

```diff
@@ -0,0 +1,20 @@
+# ab
+
+> Ferramenta de _benchmarking_ do servidor Apache. A ferramenta mais simples para realizar testes de carga.
```

Comment:
@mebeim @sbrl @agnivade How are we dealing with foreign words on translated pages? Is this the correct way to approach it?

Reply:
I havent seen anything about this on CONTRIBUTING.MD. If we decide this is the way to do it I believe this should be clearly stated there.

Reply:
I don't really see this as a problem. Some English words are ok to use in other languages, and some are better translated. There isn't really a rule to follow here, depends on the language and the specific situation. For example, "benchmarking" in Italian is a commonly used term in informatics, and I used it without problem in [the translation I made](https://github.com/tldr-pages/tldr/blob/master/pages.it/common/ab.md). Probably this is the case for Portuguese too, isn't it?

By the way, I would remove the underscores (`_`) around the word, it doesn't need any particular emphasis.

Reply:
Oh maybe I did not made clear what I was trying to say. I also don't see any problems on using english words in other languages. 

I was talking about the underscores there. Should we keep or remove them?

Reply:
@schneiderl oooh I see, I totally misunderstood haha. Regarding the underscores, I think we should remove them, there really is no need to highlight anything in that description IMHO.

Reply:
What do you mean "foreign" words? As far as I understand it, I'd recommend following any existing guidelines you know of to work out which words should be translated. How would you naturally say it in the target language?

The only things we can't really translate are the command names, anything that would break the actual command invocation, and the syntax that the linter uses to validate the page.

Reply:
Well, "benchmarking" is "benchmarking" in Italian, same for "file", "directory", and many other informatic words. What @schneiderl was talking about were the surrounding underscores (`_`), not the translation, I mixed things up with my reply :')

Reply:
Ah, the underscores mean _italics_ in markdown :P

Reply:
@sbrl yes I know, that's why I'm saying there's no need for it.

Reply:
Hi guys, I decided to put underscores(italic) because in Brazil it's mandatory on academic papers, so I'm used to doing that. 

I agree that we don't need that on TLDR